.github/workflows/typing.yml

```diff
@@ -0,0 +1,27 @@
+name: Type-check
```

Comment:
ok, we improved this wording a bit to signal that selection of initial files pretty much random. The point is to extend it.

Since this is better than nothing let's proceed